src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  position: relative;
+  top: 50%;
+  margin: auto;
+  transform: translateY(25%);
+  width: (7 * $cell-size) + 7*$gap;
+  
+  &__field {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap;
+  
+    @each $day, $value in $days {
+      &--start-day--#{$day} :first-child {
+        margin-left: ($gap + $cell-size) * ($value - 1);
+      }
+    }
+    
+    @for $day from 29 through 31 {
+      &--month-length--28 :nth-child(n + #{$day + 1}) {
```

Comment:
<img width="1391" alt="image" src="https://user-images.githubusercontent.com/66749919/217337250-a4c1e229-12f9-49f4-a2e3-1ced6d25584e.png">

Reply:
After changing day number here, last days (29, 30,31) are not hovered anymore. the function doesn`t work